pennylane_qiskit/aer.py

```diff
@@ -27,11 +27,16 @@ class AerDevice(QiskitDevice):
     Please refer to the `Qiskit documentation <https://qiskit.org/documentation/>`_ for
     for further information to the noise model and backend options.
 
-    A range of :code:`backend_options` can be given as kwargs that will be passed to the simulator.
+    A range of :code:`backend_options` that will be passed to the simulator and
+    a range of :code:`parameters` for `qiskit.compiler.transpile <https://qiskit.org/documentation/_modules/qiskit/compiler/transpile.html>`_
+    can be given as kwargs.
```

Comment:
Nice! :) One thing here is that we'd try to reference as few explicit links from Qiskit as possible. This way the docs do not have to be changed each time the link links become broken, there's an API change, etc.
```suggestion
    A range of :code:`backend_options` that will be passed to the simulator and
    a range of transpile options can be given as kwargs.
```